protos/format.proto

```diff
@@ -140,6 +140,9 @@ message DataFile {
   // Relative path to the root.
   string path = 1;
   // The ids of the fields/columns in this file.
+  //
+  // This must be equal in length to the number of columns in the file. The order
+  // of the ids determines the mapping from field id to column position.
```

Comment:
Is this correct?

Reply:
Yes, that's right. So if you already change the format.proto, let's keep a "already assigned max field id" in the manifest? To prevent re-use existing field id if deleted.